mlflow/data/huggingface_dataset.py

```diff
@@ -181,6 +181,7 @@ def from_huggingface(
     revision=None,
     name: Optional[str] = None,
     digest: Optional[str] = None,
+    trust_remote_code: Optional[bool] = None,
```

Comment:
Does `datasets` package also use `None` as the default?

Reply:
good question, the default does seem to be `None`:

https://github.com/huggingface/datasets/blob/68f67741ffde68c98d0a2f59ac4d8e3a7bc03065/src/datasets/config.py#L190-L197

if it's `None`, it will ask:

https://github.com/huggingface/datasets/blob/68f67741ffde68c98d0a2f59ac4d8e3a7bc03065/src/datasets/load.py#L113

Reply:
Got it, can we update the docstring?

Reply:
updated! i'm not sure if i should include a line like `if unspecified, this argument falls back to the datasets default`, since it's also kind of uninformative and HF might change the defaults